HHS/OPRE-OPS
# Separate Django settings for local and cloud environments; connect app to DB in cloud.gov

## Issue

Closes #158.

## Documentation added

👁️  [How to: set Django SECRET_KEY in the cloud](https://github.com/HHS/OPRE-OPS/blob/ars/django-settings/docs/recipes/set_django_secret_key.md) 👁️ 
## How to test

+ [x] Verify application code runs and connects to Postgres locally.
+ [x] Deploy app code to the cloud.gov sandbox environment. Verify application code runs and connects to Postgres in a cloud.gov sandbox environment.

To verify that the app code connects to Postgres:

Visit the app at the `/admin/` URL, and type in an arbitrary username and password. If you not allowed in, but do not get a 500 error, that means the app code is successfully talking to the Postgres database when it tries to authenticate you!
